frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx

```diff
@@ -128,25 +130,27 @@ class DatabaseEditApp extends Component {
         <Breadcrumbs className="py4" crumbs={crumbs} />
 
         <DatabaseEditMain>
-          <div>
-            <div className="pt0">
-              <LoadingAndErrorWrapper
-                loading={!database}
-                error={initializeError}
-              >
-                <DatabaseEditContent>
-                  <DatabaseEditForm>
-                    <DatabaseForm
-                      initialValues={database}
-                      isAdvanced
-                      onSubmit={handleSubmit}
-                    />
-                  </DatabaseEditForm>
-                  <div>{addingNewDatabase && <DatabaseEditHelp />}</div>
-                </DatabaseEditContent>
-              </LoadingAndErrorWrapper>
+          <ErrorBoundary errorComponent={GenericError}>
```

Comment:
Add an error boundary component so it matches the expectation in the test–that we can remove the database even if the form blows up.

<img src="https://user-images.githubusercontent.com/1937582/221267824-533037a1-1fea-47bf-ae1a-eed775085800.png" />